packages/protocol/graphics.ts

```diff
@@ -626,16 +617,6 @@ export function setPlaybackStatusCallback(callback: typeof onPlaybackStatus): vo
   onPlaybackStatus = callback;
 }
 
-let onPointsReceived: (points: TimeStampedPoint[]) => void;
-export function setPointsReceivedCallback(callback: typeof onPointsReceived): void {
-  onPointsReceived = callback;
-}
-
-let onAllPaintsReceived: (received: boolean) => void;
-export function setAllPaintsReceivedCallback(callback: typeof onAllPaintsReceived): void {
-  onAllPaintsReceived = callback;
-}
```

Comment:
Removing these listeners is a first step toward removing this module (and it's state) entirely.